docs/src/app/components/Divider/ExampleForm.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Divider from 'material-ui/lib/divider';
+import Paper from 'material-ui/lib/paper';
+import TextField from 'material-ui/lib/text-field';
+
+const DividerExampleForm = () => {
+
+  const underlineStyle = {
```

Comment:
Could you move the style outside of the function? I don't think that we need to reallocate them at each render?